pkgs/applications/networking/instant-messengers/nheko/default.nix

```diff
@@ -1,6 +1,20 @@
-{ lib, stdenv, fetchFromGitHub
-, cmake, cmark, lmdb, mkDerivation, qtbase, qtmacextras
-, qtmultimedia, qttools, mtxclient, boost, spdlog, olm, pkgconfig
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, cmark
+, lmdb
+, lmdbxx
```

Comment:
Commit message says "nheko: format inputs and arguments", but you add an unused `lmdbxx` argument.